stages/org.osbuild.systemd-journald

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python3
+"""
+Configures systemd-journald
+
+The 'config' option allows to create a systemd-journald configuration drop-in
+file in `/usr/lib/systemd/journald.conf.d` with the name `filename`.
+
+Drop-in configuration files can specify the following subset of options:
+- 'Journal' section
+  -  'Storage' option
+  -  'Compress' option
+  -  'SplitMode' option
+  -  'MaxFileSec' option
+  -  'MaxRetentionSec' option
+  -  'SyncIntervalSec' option
+  -  'Audit' option
+
+At least one option must be specified in the 'Journal' section.
+"""
+
+import configparser
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA = r"""
+"additionalProperties": false,
+"properties": {
+    "filename": {
+        "type": "string",
+        "description": "Name of the systemd-journald drop-in file.",
+        "pattern": "^([[a-zA-Z0-9_]|\\.|-]){1,250}\\.conf$"
+    },
+    "config": {
+        "additionalProperties": false,
+        "type": "object",
+        "description": "systemd-journald configuration.",
```

Comment:
```suggestion
        "description": "systemd-journald configuration",
```

(We only end proper sentences with a full-stop (at least a verb somewhere in there), but avoid it for other types of annotations.)